packages/ckeditor5-core/README.md

```diff
@@ -10,7 +10,7 @@ This package implements CKEditor 5's core editor architecture &mdash; a set of c
 
 ## Documentation
 
-For general introduction see the [Overview of CKEditor 5 framework](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/overview.html) guide and then the [core editor architecture guide](https://ckeditor.com/docs/ckeditor5/latest/framework/guides/architecture/core-editor-architecture.html).
```

Comment:
This address (https://ckeditor.com/docs/ckeditor5/latest/framework/guides/overview.html) is actually redirected to https://ckeditor.com/docs/ckeditor5/latest/framework/index.html, which means the new version no longer works. We need to add a new redirect or change this to the actual link.